README.md

```diff
@@ -4,85 +4,60 @@
 
 # ESP-IDF VS Code Extension
 
+[![Forum](https://img.shields.io/badge/Forum-esp32.com-blue)](https://esp32.com/viewforum.php?f=40)
+
 Develop and debug applications for Espressif [ESP32](https://espressif.com/en/products/hardware/esp32), [ESP32-S2](https://www.espressif.com/en/products/socs/esp32-s2) chips with [ESP-IDF](https://github.com/espressif/esp-idf) IoT Development Framework.
```

Comment:
Also this statement feels redundant, maybe remove this and add supported chips as `badge` after `forum`

[![ESP32-S2](https://img.shields.io/badge/Supported%20Chip-ESP--32,%20ESP32--S2-green)](https://espressif.com/en/products/hardware/esp32-s2)

Also in the link of this badge add another doc file with list of supported targets, boards and framework (like MDF, Arduino etc)